### DISCUSSION TITLE
Wie kann man Artikel individuell sortieren?

### DISCUSSION THREAD

**Discussion by danspringer** [FriendsOfREDAXO/.github#22] [Q&A] [answered]
Ich würde gerne im Frontend eine individuelle Sortierung der Artikel (abweichend von der im Backend vorhandenen Artikel-Priorisierung) vornehmen.
Z.B. würde ich gerne die Artikel alphabetisch sortieren.

Die Artikel hole ich mir mit der folgenden Redaxo-Methode:

`$articles = rex_category::get("REX_CATEGORY_ID")->getArticles(true);`

**Comment by danspringer** [comment, marked as answer]
Für mich hat folgendes für eine Sortierung nach Namen wie gewünscht funktioniert:

```
$articles = rex_category::get("REX_CATEGORY_ID")->getArticles(true);
usort($articles, function ($a, $b) {
    return strcmp($a->getValue('name'), $b->getValue('name'));
    }
);
```

**Comment by tbaddade** [comment]
Ergänzend dazu 
https://github.com/yakamara/yakme/blob/master/lib/Yakme/Sort.php
